test/testcrawl.js

```diff
@@ -47,6 +47,41 @@ describe("Test Crawl", function() {
         });
     });
 
+    it("should obey rules in robots.txt", function(done) {
+
+        this.slow("2s");
+
+        var crawler = new Crawler("127.0.0.1", "/", 3000);
+        crawler.start();
+
+        crawler.on("fetchcomplete", function(queueItem) {
+            queueItem.url.should.not.equal("http://127.0.0.1:3000/forbidden");
+        });
+        crawler.on("complete", function() {
+            done();
+        });
+    });
+
+    it("should be able to disregard rules in robots.txt", function(done) {
+
+        this.slow("2s");
```

Comment:
Is it taking two seconds? (I'm just worried that if we get too many of these slow tests they'll all compound to create a test suite that takes ten years to run, haha.)